### PR TITLE
more loglevel info

### PR DESCRIPTION
**Pull request by markhu**
adds log.Info to address issue #151

### CODE REVIEW

**Comment by viktorbenei** [comment]
Awesome idea @markhu ! Don't forget to address @godrei 's change request ;)

File: cli/add.go [hunk 1]
```diff
@@ -213,6 +213,8 @@ func add(c *cli.Context) error {
 
 	if err := addEnv(key, value, expand, replace, skipIfEmpty); err != nil {
 		log.Fatal("[ENVMAN] Failed to add env:", err)
+	} else {
+		log.Infof("[ENVMAN] added %s: %s", key, value)
```
Comment (markhu) [inline cli/add.go]: A single log line.

**Comment by trapacska** [comment]
Hey @markhu 👋 

Just dig myself into this change and I have couple of concerns about it.
- When you export a key:value pair for example bash, you won't have confirmation message about the successful state, could you please describe why is it required to be printed at info log level?
- In the `add` subcommand we don't print anything at info level, having a new log level type wired in the `add` subcommand would mean a breaking change for those who rely on the log to be empty in case of success.
- It can also cause multiple security issues as well, in couple of cases users are exporting their secret values into ENVs by envman, exposing these values are not acceptable in the default log level(which is info), it should take separated action to log these out. (I suggest it to be a debug log type)

Thanks for your response!
🚀

**Comment by markhu** [comment]
The `info` log level is not normally visible. So the default behavior addresses all your concerns.

**Comment by trapacska** [comment]
The default log level is info. 🙂 
screen shot 2018-12-11 at 9 20 43 am

**Comment by trapacska** [comment]
@markhu So could you please move this log into a higher log level?

**Comment by trapacska** [comment]
Hey @markhu !

Closing the issue as we didn't receive response about the review. Feel free to re-open or contact us any time of you want to move forward with it! 🙂 

Happy Building! 🚀

**Pull request closed without merging**